web-jehlomat/src/Components/Inputs/DateTimeInput/TimeInput.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC, InputHTMLAttributes, useState } from 'react';
+import { default as MInput } from '@mui/material/TextField';
+import styled from '@emotion/styled';
+import DateAdapter from '@mui/lab/AdapterDayjs';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import TimePicker from '@mui/lab/TimePicker';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+    name?: string;
+    label?: string;
+    error?: string | undefined;
+    fullWidth?: boolean;
+    className?: string;
+    width?: string;
+    undertext?: string;
+}
+
+const Container = styled.div`
+    width: 100%;
+    position: relative;
+    border-radius: 90px;
+    position: relative;
+    width: 90px;
+`;
+
+const DateTimeUndertext = styled.label`
```

Comment:
Most of the components are identical with components from Date picker, consider move them to own file and use them on both places, they're over all reusable component